lib/bridge/pbn.rb

```diff
@@ -1,7 +1,5 @@
 #todo: fix this; eliminate usage of frowned-upon autoload; make dependencies explicit and just-in-time?
-module Bridge
-
-end
+module Bridge end
```

Comment:
@danielevans I briefly tried require_relative '../bridge.rb' here, placing the empty module there, but that caused problems when I wanted to depend over in bridge-stats both on the bridge gem and on the bridge-pbn gem.  I think what happened was that the bridge-pbn's empty bridge.rb was preferred for the require 'bridge' command in bridge-stats. I feel as though I'm missing something: there ought to be a simple way, without "module this module that module foo module bar ... end end end end" to quietly control a child namespace without interfering with its parent.